okhttp-testing-support/src/main/kotlin/okhttp3/RecordingEventListener.kt

```diff
@@ -129,7 +129,7 @@ open class RecordingEventListener : EventListener() {
     for (lock in forbiddenLocks) {
       assertThat(Thread.holdsLock(lock))
           .overridingErrorMessage(lock.toString())
-          .isFalse()
+          .isFalse
```

Comment:
We've pushed back on this suggestion previously, and are consistent in use of isFalse()

Reply:
Reverted.